AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift

```diff
@@ -83,6 +84,7 @@ struct TickerIdFilter {
         case .binance_smart_chain: return true
         case .avalanche: return true
         case .polygon: return true
+        case .candle: return true
```

Comment:
Would you help make this false? Rationale like above (about CoinGecko)

Reply:
Fixed